Sources/Lifecycle/Lifecycle.swift

```diff
@@ -12,6 +12,10 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if compiler(>=5.5)
+import _Concurrency
```

Comment:
Is this import required?

Reply:
It'll be gone eventually (it should be already but seems something went wrong)